package.json

```diff
@@ -2,7 +2,25 @@
   "name": "single-spa-react",
   "version": "2.14.0",
   "description": "A single spa plugin for React apps",
-  "main": "lib/single-spa-react.js",
+  "main": "lib/umd/single-spa-react.js",
+  "module": "lib/esm/single-spa-react.js",
+  "type": "module",
+  "exports": {
+    ".": {
+      "import": "./lib/esm/single-spa-react.js",
+      "require": {
+        "browser": "./lib/umd/single-spa-react.js",
```

Comment:
When would `require` be used in a browser setting? Bundlers like webpack/rollup have support (and even prefer) using ESM over CJS for dependencies.

I think this can be removed?

Reply:
Ok